var/spack/repos/builtin/packages/py-umi-tools/package.py

```diff
@@ -27,3 +27,4 @@ class PyUmiTools(PythonPackage):
     depends_on("py-regex", type=("build", "run"))
```

Comment:
True enough. It's still in the `requirements.txt` but backtracking it, it hasn't been in the `setup.py` since `@0.4.4` so was never needed in the first place.